b288702/keyring.asc

```diff
@@ -1,6 +1,6 @@
 Created with running:
 
-    $ curl --silent https://raw.githubusercontent.com/filecoin-project/phase2-attestations/9014d0d57a03f9eb68297d9ee68994fb238b26e7/b288702/README.md|grep '^\['|cut -d ' ' -f 2|xargs -n 1 sh -c 'echo $0 && curl --silent --location $0|gpg --no-default-keyring --keyring ./keyring.gpg --import'
+    $ curl --silent https://raw.githubusercontent.com/filecoin-project/phase2-attestations/e9ec1d11ffe0a98b95199833b3ebd537e589bcbe/b288702/README.md|grep '^\['|cut -d ' ' -f 2|xargs -n 1 sh -c 'echo $0 && curl --silent --location $0|gpg --no-default-keyring --keyring ./keyring.gpg --import'
```

Comment:
Please don't change this, as it isn't true. It should mention the commit it was actually created with. Also the checksum won't match the one uploaded to S3 then.

Reply:
Good point, this is one thing I did not verify when I was checking the GPG workflow. Will force-push without this change.

Reply:
@vmx removed https://github.com/ribasushi/phase2-attestations/commit/d832cd5809941054ec entirely, will redo the doc portion when writing the final prose for it once the process is final.